app/guide/using-markdown.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title: Using Markdown
+description: A guide to using Markdown within design history posts
+eleventyNavigation:
+  parent: Guide
+---
+
+Posts within this design history website are written using [Markdown](https://en.wikipedia.org/wiki/Markdown), a simple markup language which saves you from having to use HTML.
+
+## Paragraphs
+
+You don’t have to do anything special for paragraphs, except for making sure that there is an empty line between each one.
+
+```md
+This is a paragraph.
+
+This is another paragraph.
+This sentence will appear next to the previous sentence as single line breaks are ignored.
+```
+
+## Headings and sub-headings
+
+For headings, start the line with hash (`#`) characters, followed by a space.
+
+You should use 2 hash characters for a heading (h2), or 3 hash characters for a sub-heading (h3).
+
+Do not using a single hash character for the post title (h1) as this will be generated automatically.
+
+You do not need to create a heading before the introductory paragraph.
+
+```md
+This is an introductory paragraph.
+
+## Heading
+
+This is a paragraph.
+
+### Sub-heading
+
+This is a paragraph.
+```
+
+## Links
+
+To create links, you have to use a combination of square brackets and rounded brackets.
```

Comment:
I think colloquially these are round brackets, not rounded brackets.

Reply:
done